### DISCUSSION TITLE
Add option to disable "prefix" pop-up (Pending Key Sequence)

### DISCUSSION THREAD

**Discussion by maxRN** [ghostty-org/ghostty#3866] [Feature Requests, Ideas]
Currently when you configure a keybind with a [prefix](https://ghostty.org/docs/config/reference#keybind) you get a "pending key sequence" pop up at the bottom of the terminal window: 
<img width="325" alt="image" src="https://github.com/user-attachments/assets/9b153b00-6e9e-4532-addb-85e373ae48e1" />
 
 I can see how it's helpful so you don't forget what mode you are in, but personally I find it a bit distracting and would like to disable it.

**Comment by oliver-cfc** [comment]
I find it distracting at the bottom because my cursor is normally at the bottom too so it covers the current line.  I kind of like seeing it though so it would great if it could be configurable to be top/bottom/off.

**Comment by godexsoft** [comment]
I find it very distracting too. It can be useful on a standard keyboard (e.g. on a laptop) but i mostly use a highly configurable keyboard where i have binds for all sort of tmux-style pane/split management. In such case it really is useless to show the panel
